### DISCUSSION TITLE
change username account get error

### DISCUSSION THREAD

**Discussion by azadi-behzad** [community/community#28468] [Other Feature Feedback, Questions, & Ideas]
hi 
i want to change my username account github
but get me error "The repository [name] has been retired and cannot be reused"

my username is behzad-azadi2693
i want change to behzad-azadi

please help

**Comment by billythekid** [comment]
Hi @behzad-azadi2693 

Looks like you're running into our [Popular repository namespace retirement](https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement) feature. 

**If you're 100% sure** you're the proper owner of the username/repo combination (the namespace) that's been retired, you'll need to [raise a support ticket](https://support.github.com/contact/account?subject=problem+with+retired+namespaces&type=account) about this as we may need to discuss private details about your account history with you that we can't discuss on the public forum. 

Best of luck!
Billy 🤠
